gcp_variant_transforms/transforms/extract_input_size.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A PTransform to output a PCollection of ``Variant`` records to BigQuery."""
+
+from __future__ import absolute_import
+
+import apache_beam as beam
+
+from apache_beam.io import filesystems
+from apache_beam.typehints import Any, List, Iterable
+from apache_beam.typehints import with_input_types
+from apache_beam.typehints import with_output_types
+from gcp_variant_transforms.beam_io import vcf_estimate_io
+
+
+@beam.typehints.with_input_types(vcf_estimate_io.VcfEstimate)
+class ExtractLineCount(beam.DoFn):
+  def process(self, estimate):
+    yield estimate.estimated_line_count
+
+
+@beam.typehints.with_input_types(vcf_estimate_io.VcfEstimate)
+class ExtractFileSize(beam.DoFn):
+  def process(self, estimate):
+    yield estimate.size_in_bytes
+
+
+@with_input_types(List[int])
+@with_output_types(int)
+class SumEstimations(beam.CombineFn):
```

Comment:
Is it just summing together the elements in all PCollections? You can use beam.CombineGlobally(sum) directly. There are some other basic commonly used transforms in beam, for instance, https://github.com/apache/beam/blob/master/sdks/python/apache_beam/transforms/combiners.py